desktop/cmp/treemap/SplitTreeMapModel.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+import {HoistModel, managed} from '@xh/hoist/core';
+import {bindable, computed} from '@xh/hoist/mobx';
+import {throwIf} from '@xh/hoist/utils/js';
+import {isNil, isFunction, sumBy} from 'lodash';
+
+import {TreeMapModel} from './TreeMapModel';
+
+/**
+ * Todo
+ */
+@HoistModel
+export class SplitTreeMapModel {
+
+    //------------------------
+    // Immutable public properties
+    //------------------------
+    /** @member {GridModel} */
+    gridModel;
+    /** @member {function} */
+    filter;
+    /** @member {Object} */
+    treeMapModelConfig;
+    /** @member {string} */
+    orientation;
+
+    /** @member {TreeMapModel} */
+    @managed posModel;
+    /** @member {TreeMapModel} */
+    @managed negModel;
+
+    //------------------------
+    // Observable API
+    //------------------------
+    /** @member {Object} */
+    @bindable.ref config = {};
+
+    @computed
+    get posRootTotal() {
+        return sumBy(this.posModel.data, it => {
+            // Only include root records that pass the filter
+            if (it.parent || !this.filter(it.record)) return 0;
+            return it.value;
+        });
+    }
+
+    @computed
+    get negRootTotal() {
+        return sumBy(this.negModel.data, it => {
+            // Only include root records that don't pass the filter
+            if (it.parent || this.filter(it.record)) return 0;
+            return it.value;
+        });
+    }
+
+    /**
+     * @param {Object} c - SplitTreeMapModel configuration.
+     * @param {Object} c.config - Highcharts configuration object for the managed chart. May include
```

Comment:
is the 'config' name consistent with something else?  Seems like it would be useful to have highCharts in the name here...